aseprite.sh

```diff
@@ -3,7 +3,7 @@
 # Script to automate building latest release of Aseprite (it can be release or beta build)
 # This is for macOS build version.
 
-POSTFIXPATH_SDKROOT=Platforms/MacOSX.platform/Developer/SDKs/MacOSX11.3.sdk
+POSTFIXPATH_SDKROOT=Platforms/MacOSX.platform/Developer/SDKs/MacOSX.sdk
```

Comment:
It should be fine to use the current installed SDK, rather than requiring an old one...

Reply:
I'm skeptical about this as it relates to #11 , I think we should not touch this for now.